src/components/dapp-staking/modals/RegisterDappMedia.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div>
+    <q-input
+      :model-value="data.videoUrlInput"
+      outlined
+      label="Youtube Video link"
+      maxlength="500"
+      :rules="[
+        (v) => (v && v.length > 0) || 'dApp video link is required.',
+        (v) => getVideoId(v) !== null || 'Enter a valid YouTube url',
+      ]"
+      class="tw-my-2"
+      @update:model-value="videoUrlChanged"
+    >
+      <template #prepend>
+        <q-icon name="movie" />
+      </template>
+      <template v-if="data.videoUrl" #append>
+        <q-icon
+          v-if="data.videoUrl"
+          name="remove_red_eye"
+          class="tw-cursor-pointer"
+          @click="toggleViewPreview()"
+        />
+      </template>
+    </q-input>
+    <q-video v-if="showVideoPreview" :src="getEmbedUrl(data.videoUrl)" />
+    <q-file
+      v-model="data.images"
+      outlined
+      multiple
+      append
+      counter
+      max-file-size="102400"
```

Comment:
I am not sure about the database side so I won't comment a lot, but I think 100kb is too small for images? I think it should be maximum of 3mb

Reply:
I think 3MB is too much. We will have min 4 images which is at least 12MB downloaded just for displaying them in carousel. I enabled option to have a full screen carousel, but still think that 3MB per image is too much. Maybe 1MB?

Reply:
You're right, since we don't have SSR yet, 1mb seems reasonable